README.md

```diff
@@ -1,71 +1,59 @@
+```bash
+pip install -e .
+```
 
-With a meta-manifest file created, conda-vendor can then create local channels. 
-	
-	conda vendor channels --meta-manifest-path ./meta_manifest.yaml
+Running Tests:
+```bash
+pytest tests/ -vvv -s
+```
 
-This will create a directory called `local_channel` that will contain the same number of channels as were listed in the original `environment.yaml` file. These local channels will only contain the packages that are needed to satisfy the solved environment from the `meta-manifest` step.
-
-### Using the Local channel
-
-There are several ways to use the local channel. If python was in the input `environment.yaml` file for example, the following could be used:
-
-	conda create -n test_env python -c <path_to_local_channel> --offline
-	
-The `--offline` flag will prevent conda from reaching out to the internet for packages. To verify that the environment created only contains packages contained in the local channel, run the following:
-
-	conda activate test_env
-	conda list --explicit
-	
-This should show a list of all the packages in the environment the local paths to their source code (typically tar.bz2 files).
-
-### Creating Environment with all Packages from Input Environment.yaml
-
-To generate a conda environment yaml that contains all the packages from the input `environment.yaml`, run the following:
+## Usage
 
-	conda vendor local-yaml --meta-manifest-path ./meta_manifest.yaml --channel-root <absolute_path_to_local_channel_dir>
-	
-This will create a environment file inside the `local_channel` directory called `local_conda-vendor-env.yaml`. An environment can then be created with:
+#### Supported Solvers
+* Conda
+* Mamba
+* Micromamba
 
-	conda env create -f local_channel/local_conda-vendor-env.yaml
-	
-The environment will be created with the packages that are contained in the local channel.
+Vendor dependencies from an  `environment.yaml` into a local channel:
+```bash
+# Use conda as the solver for linux-64
+conda-vendor vendor --file environment.yaml --solver conda --platform linux-64
 
-### Creating a Custom Manifest for Package Security Validation
+# use mamba as the solver for osx-64
+conda-vendor vendor --file environment.yaml --solver mamba --platform osx-64
 
-The following functionality is only applicable if there is an organization that requires a list of packages for security validation. Currently the Iron Bank format is supported, but support for other formats can be added to the source code in `custom_manifest.py`.
+# use micromamba as the solver for the host platform
+conda-vendor vendor --file environment.yaml --solver micromamba
+```
 
-To generate an iron bank manifest from the meta-manifest, run:
+Use Dry-Run install to verify that conda can solve using only the vendored channel:
+```bash
+# NOTE: ensure to use the same solver used to create the vendored channel
+# example channel vendored with mamba: that includes micromamba python and pip
```

Comment:
I guess we decided to always include pip because conda has it as its default.

But if the user wanted a non pip environment they would need `CONDA_ADD_PIP_AS_PYTHON_DEPENDENCY=False` which we can ignore if pip is just always part of the vendored packages.